aunt-leahs-app/src/components/table/TableComponent.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
I think you meant to delete the above TableComponent.js file and replace it with this TableComponent.jsx file.

Reply:
Yes... I think I deleted it but I do not know why git recreated it.